sdk/src/main/java/co/omise/android/ui/CreditCardActivity.kt

```diff
@@ -118,20 +114,22 @@ class CreditCardActivity : AppCompatActivity() {
         val expiryYear = expiryDateEdit.expiryYear
         val securityCode = securityCodeEdit.securityCode
 
-        val request = Token.CreateTokenRequestBuilder(
-                name,
-                number,
-                expiryMonth,
-                expiryYear,
-                securityCode
-        ).build()
+        val cardParam = CardParam(
```

Comment:
I mean in other place in the project.

Reply:
I'm not sure, is there is any other place where you feel the same pattern needs to be applied? I had to do it this way because I couldn't get the params to be format the way I wanted them. This way, CardParam becomes its own `Json` and is accepted by the API.

Reply:
e.g. CreateSourceRequestBuilder. I know you want to make it as `Json` payload. But for the java user I'm not sure it'll look good or not.
How's about having a builder to help user build `CardParam`? What do you think?

Reply:
We can make it into a builder, but 5 out 7 fields here would still need to be included in the constructor as they are mandatory right?

Reply:
I not sure, but I think we should keep it consistent if we choose the constructor pattern we need to use that all of params. What do you think?

Reply:
I'm not sure I understand 🤔

Reply:
Then how's Source create parameters?

Reply:
Same way, `Source` serializes all its params on the base json 

`
{
"something" : "SOMETHING",
"something_else" : "SOMETHING_ELSE"
}
`

But in Token we needed something like this 
`
{
"card":
{
"name":"abc",
"number":"4242424242424242",
"expiration_month":1,
"expiration_year":2020,
"security_code":"123"
}
}
`

So that's where the difference comes from.

Reply:
I mean, do we use the constructor pattern to build Source's parameter?

Reply:
Mix of both, mandatory fields in constructor and others as Builder. In Token, we have just `Card` as a mandatory one in the constructor.

Reply:
OK, then approved. 😂

Reply:
:)